packages/devtools/README.md

```diff
@@ -7,13 +7,13 @@ Developer tooling, used by statechannels packages.
 From a node project, install as a dev dependency:
 
 ```
-$ npm i -D https://github.com/magmo/devtools
+$ npm i -D https://github.com/statechannels/monorepo/blob/master/packages/devtools
 ```
 
 or
 
 ```
-$ yarn add -D https://github.com/magmo/devtools
+$ yarn add -D https://github.com/statechannels/monorepo/blob/master/packages/devtools
```

Comment:
Do these commands actually work?

Reply:
I am not sure... I didn't actually test them :)

Should we even teach how to install this as its internal?

Reply:
I tried them: they don't work. I would just remove this section for now.